dev-python/pydotplus/pydotplus-2.0.2.ebuild

```diff
@@ -1,38 +1,29 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-PYTHON_COMPAT=( python3_6 )
+PYTHON_COMPAT=( python3_{6,7,8,9} )
 
 inherit distutils-r1
 
 DESCRIPTION="Improved version of the old pydot project"
-HOMEPAGE="http://pydotplus.readthedocs.org/"
+HOMEPAGE="https://pydotplus.readthedocs.org/"
 SRC_URI="mirror://pypi/p/${PN}/${P}.tar.gz"
 
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
-
-DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
-#	test? (
-#		dev-python/flake8[${PYTHON_USEDEP}]
-#		dev-python/pytest[${PYTHON_USEDEP}]
-#		dev-python/pytest-cov[${PYTHON_USEDEP}]
-#		dev-python/sphinx[${PYTHON_USEDEP}]
-#		dev-python/sphinx_rtd_theme[${PYTHON_USEDEP}]
-#		dev-python/tox[${PYTHON_USEDEP}]
-#		)
+
+DEPEND=""
 RDEPEND="
 	dev-python/pyparsing[${PYTHON_USEDEP}]
 	"
 
-#test phase curently disabled, waiting on upstream to include
-#the required files:
-#https://github.com/carlos-jenkins/pydotplus/issues/12
-#python_test() {
-#	${EPYTHON} -m unittest discover || die
-#	tox
-#}
+PATCHES=( "${FILESDIR}/${P}-tests.patch" )
+
+python_test() {
+	pushd test > /dev/null || die
+		python pydot_unittest.py || die
+	popd > /dev/null
```

Comment:
||die

Reply:
most ebuilds have it without die (232 without, 217 with):

```
chymera@silenthost ~/src/pydotplus $ qgrep "popd > /dev/null" | wc -l
449
chymera@silenthost ~/src/pydotplus $ qgrep "popd > /dev/null || die" | wc -l
217
```

Why is this required here? doesn't the dir just change to the correct one after the function exits anyway?